.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,43 @@
+# Proposed change
+
+<!--- 
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+--->
+
+Fixed issues: <!--insert issue reference in this form: #XX #XX-->
```

Comment:
This will not auto-link those issues for closure. You need the format "Fix/fxes/fixed" or similar.